### DISCUSSION TITLE
使用MVVM工具包的source generator功能，VS的 intellisense提示稀烂

### DISCUSSION THREAD

**Discussion by TrmorySpindrift** [BYJRK/DotNet-Discussions#115] [WPF 开发]
最近开发wpf工具，尝试了使用Mvvm工具包的source generator功能，少写很多boilerplate 代码感觉很爽。
但是我发现我的vs代码提示直接稀烂，比如打上[ObservableProperty]属性的field，对应的Property名全是下划线，无法提示
我是大番茄和vs自带的intellisense都开了，代码提示和补全都无效，有解决办法吗？

VS版本17.5.4

**Comment by SagapoSheepL** [comment]
这个看运气的，一般情况下你写的语法没有问题就可以正常提示，但是一旦写的有问题生成器就会报错，然后所有的自动生成的所有东西都会报错，这个时候检查一下有没有写错，有时候特性和字段先后顺序也会导致报错，这时候要么生成一下项目可能会好，不行就得重启VS。我目前没有发现有效方案，一般都是不理会报错，直接运行，在生成窗口会提示真正报错的语句。

**Comment by CodingOctocat** [comment]
没办法，官方估计修复了好几次，问题依旧。

**Comment by TrmorySpindrift** [comment]
好吧，那就不管了，能少写废话代码我知足了
